### PR TITLE
Added option to write out data in stokes I only instead of IQUV

### PR DESCRIPTION
**Pull request by chiamintan**
I think these are all I need to change to make things work. As far as I can see, the nstokes parameters only affects what kind of output do we want to write into the psrfits files. I made them take in the value input by the tied array beam script instead of a default value in the include file.

### CODE REVIEW

File: app/make_mwa_tied_array_beam.c [hunk 1]
```diff
@@ -568,6 +574,15 @@ void make_tied_array_beam_parse_cmdline(
                 case 'n':
                     opts->nchunks = atoi(optarg);
                     break;
+                case 'N':
+                    opts->out_nstokes = atoi(optarg);
+                    if ((opts->out_nstokes != 1) && (opts->out_nstokes != 4))
```
Comment (bwmeyers) [inline app/make_mwa_tied_array_beam.c:579]: I think this should be an "OR" rather than "AND"?

**Review by bwmeyers — COMMENTED** [review]
I think this looks correct, barring the one comment regarding logic operators.

File: app/make_mwa_tied_array_beam.c (hunk 1 above)
Comment (chiamintan) [reply app/make_mwa_tied_array_beam.c:579]: I started with "OR" and realised I need both of these to be true actually, so "AND" is the right operator.

Comment (bwmeyers) [reply app/make_mwa_tied_array_beam.c:579]: Oh yes. I misread it, but for the case where you want it to FAIL then it's correct. "If it's not 1 and not 4, those are the only allowed options, so fail."

**Review by bwmeyers — APPROVED** [review]

**Comment by chiamintan** [comment]
should I merge?

**Comment by bwmeyers** [comment]
Sorry, I meant to do that yesterday. :)

**Pull request merged**